lib/modules/manager/github-actions/extract.ts

```diff
@@ -1,21 +1,23 @@
 import is from '@sindresorhus/is';
 import { load } from 'js-yaml';
 import { logger } from '../../../logger';
+import { isNotNullOrUndefined } from '../../../util/array';
 import { newlineRegex, regEx } from '../../../util/regex';
+import { GithubRunnersDatasource } from '../../datasource/github-runners';
 import { GithubTagsDatasource } from '../../datasource/github-tags';
 import * as dockerVersioning from '../../versioning/docker';
 import { getDep } from '../dockerfile/extract';
 import type { PackageDependency, PackageFileContent } from '../types';
 import type { Workflow } from './types';
 
-const dockerActionRe = regEx(/^\s+uses: ['"]?docker:\/\/([^'"]+)\s*$/);
-const actionRe = regEx(
+const dockerActionRegex = regEx(/^\s+uses: ['"]?docker:\/\/([^'"]+)\s*$/);
+const actionRegex = regEx(
```

Comment:
I thought it would be good to use good straight-forward names, so you don't have to guess what a variable means.

Cleaning up as you go seems okay to me. But maybe the maintainers want to rename the variable in the source file?

I'll defer to the Renovate maintainers on this matter.

I don't mean for my review comments to annoy you or to slow down the merge process.

Reply:
I agree, `Regex` is more readable than `Re`. Your review comments were helpful to improve the quality of the PR. Also, your explanations are detailed enough to understand the intention of the change request.  👍 Thanks, @HonkingGoose!

Reply:
We don't like to have unrelated changes on feature or bugfix PR's. Those changes should always be on separate PR's. Especially if they are so many, they make review a lot harder than necessary.

Reply:
Alright, I have renamed the variables to its original names since I don't see another possibility to get this PR merged. 
I don't want to overwhelm the maintainers with 8 additional lines of changed code.

In the future, I will consider other open source projects which align more with my software development ideas.